src/pywrapper.cpp

```diff
@@ -290,6 +291,17 @@ bool PyWrapper::convert(void* in_, MultiTypeValue& out)
                 out.vf.push_back(val);
                 out.type = MultiTypeValue::Type::VECTOR_FLOAT;
             }
+	    if(PyBytes_Check(el) && (out.type == MultiTypeValue::Type::NONE || out.type == MultiTypeValue::Type::VECTOR_STRING ||
+				     out.type == MultiTypeValue::Type::VECTOR_FLOAT /* how not to end here ? */
+				     )) {
+	        const char *cval = PyBytes_AsString(el);
+	        if (!cval && PyErr_Occurred()) {
+                    PyErr_Clear();
+                    return false;
+                }
+	        out.vs.push_back(cval);
```

Comment:
What happens if cval==NULL

Reply:
changed. cval is the return value. Implies PyErr_Occured to true according to documentation.
Defensive check